engines/e_dasync.c

```diff
@@ -838,7 +842,7 @@ static int dasync_rsa_paramgen(EVP_PKEY_CTX *ctx, EVP_PKEY *pkey)
 
     if (pparamgen == NULL)
         EVP_PKEY_meth_get_paramgen(dasync_rsa_orig, NULL, &pparamgen);
-    return pparamgen(ctx, pkey);
+    return pparamgen != NULL ? pparamgen(ctx, pkey) : 1;
```

Comment:
We've only don the init functions elsewhere. Not sure why paramgen is special to have both the init and the main function handled?

Reply:
RSA doesn't imply any paramgen (and callback) so we should succeed (and not get a segfault)